src/io/flutter/run/LauncherState.java

```diff
@@ -50,28 +50,28 @@
 
 /**
  * Launches a flutter app, showing it in the console.
- *
- * <p>Normally creates a debugging session, which is needed for hot reload.
+ * <p>
+ * Normally creates a debugging session, which is needed for hot reload.
  */
-public class Launcher extends CommandLineState {
+public class LauncherState extends CommandLineState {
```

Comment:
Okay how about LaunchState? (Slightly shorter.)

(I think maybe I can avoid inheriting from CommandLineState, but that's a refactoring for another time.)